src/styles/styles.js

```diff
@@ -2874,6 +2874,37 @@ const styles = {
         paddingTop: 80,
         paddingBottom: 45,
     },
+
+    joinRoomPromptContainer: {
+        maxHeight: 92,
+        height: 92,
+        borderRadius: 18,
+        padding: 20,
+        backgroundColor: themeColors.overlay,
+        display: 'flex',
+        flexDirection: 'row',
+    },
+
+    joinRoomPromptTextContainer: {
+        paddingTop: 3.5,
```

Comment:
What does 3.5 do exactly?

Reply:
When I was in the Figma file I noticed the text had some extra padding (in addition to the 20px given by the outer container) that wasn't being applied to the other elements. But thinking about it now i can probably achieve the same effect with flexbox, will change!